server/tests/api/single-pod.js

```diff
@@ -187,14 +187,19 @@ describe('Test a single pod', function () {
   })
 
   it('Should have the views updated', function (done) {
-    videosUtils.getVideo(server.url, videoId, function (err, res) {
-      if (err) throw err
+    this.timeout(5000)
 
-      const video = res.body
-      expect(video.views).to.equal(1)
+    setTimeout(videosUtils.getVideo,
```

Comment:
Please move the setTimeout in the previous tests (and try with 500ms):

```
 videosUtils.testVideoImage(server.url, 'video_short.webm', video.thumbnailPath, function (err, test) {
        if (err) throw err
        expect(test).to.equal(true)

        // Wait the async views increment
        setTimeout(done, 500)
})
```

So you don't need to modify the `Should have the views updated` test.